wfc_gh/WaveFunctionCollapse.cs

```diff
@@ -123,8 +120,7 @@ protected override void SolveInstance(IGH_DataAccess DA)
                                                            adjacencyRulesModuleLow.Count),
                                                   adjacencyRulesModuleHigh.Count);
 
-            if (adjacencyRulesMinCount == 0)
-            {
+            if (adjacencyRulesMinCount == 0) {
```

Comment:
actually at least 3 except small worlds where 0, 1 and 2 suffice

I understand that this relates to 291 but for the user the above is a more useful information

Reply:
You are right, This only makes sense with at least 3 rules. I'll update it/

Reply:
Nevermind! For 1D worlds, this limit is actually 1, and for 2D worlds it is two. Keeping as is.